packages/story-editor/src/app/story/storyTriggers/storyEvents/propTypes.js

```diff
@@ -17,7 +17,7 @@
  * External dependencies
  */
 import PropTypes from 'prop-types';
-import { OutputStory as storyPropType } from '@googleforcreators/output'; // @todo See issue #11015
+import { OutputStory as storyPropType } from '@googleforcreators/output';
```

Comment:
Why remove the todo? This still is incorrectly using a React component as a prop type.

Reply:
Oh yep I missed that.